paper/basic_training.tex

```diff
@@ -95,25 +100,35 @@ \section{Introduction}
+%\todo[inline, color={red!20}]{Put in a few-sentence discussion re the size \& timescales of systems and the appropriate method; perhaps like the images often in papers; what are typical sizes and timescales that are tractable? This of course changes with time, but this is a living document so we're good. Also add in general, that the topology does not change--most FF do not allow chemical reactions}
+
+Speed is a particular concern when describing condensed phase systems, as we are often interested in the properties of molecules (even biomacromolecules) in solution, meaning that systems will consist of thousands to hundreds of thousands or millions of atoms.
+While system size alone does not require a classical description, if we are interested in calculations of thermodynamic properties like free energy at finite (often laboratory) temperatures, these include substantial entropic contributions (as further discussed below) meaning that fluctuations and correlations of motions within the system affect computed properties, meaning that simulations must not just sample single optimal states but instead must sample the correct distribution of states -- requiring simulations of some length.
+Furthermore, many systems of interest, such as polymers (biological and otherwise) have slow motions that must be captured for accurate calculation of properties.
+For example, for proteins, relevant timescales span from nanoseconds to seconds or more, and even rearrangements of buried amino acid sidechains can in some cases take microseconds or more, with larger conformational changes and protein folding taking even longer~\cite{Schlick:2010:, Mobley:2012:JComputAidedMolDes}. 
+Recent hardware innovations have made microsecond-length simulations for biological systems of 50-100,000 atoms relatively routine, and herculean efforts have pushed the longest simulations out past the millisecond range, but the field would like to reach even longer timescales, meaning that switching to a more detailed energy model is only done with some trepidation slower energy evaluations mean less time available for sampling. 
+Thus the need for speed limits the use of quantum mechanical descriptions.
+
+Thus, for the rest of this document we will restrict ourselves to classical MD.
 
-For the rest of this document we will restrict ourselves to classical MD.
+One other important note is that, within classical molecular simulations, bond breaking and forming is generally not allowed (with notable exceptions such as reactive force fields),  meaning that the overall topology or chemistry of a system will remain constant as a function of time.
+That is, the particles comprising the system move around, but its chemical identity is a constant over the course of the simulation.
```

Comment:
This is somewhat unclear.  You seem to refer to the "chemical identity" of the entire system, which to me seems a little ambiguous.  With stoichiometrically balanced chemical reactions, the overall atomic composition of the system could stay constant, but the molecular species would be different.  If we're talking mainly about making and breaking bonds, we should maybe say that the chemical identity (including overall charge, etc.) of each _molecule_ in the system is a constant in time.